src/components/Answer/Answer.jsx

```diff
@@ -56,6 +56,10 @@ function Answer({ children, image, username }) {
           </div>
         </div>
         {children}
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Dolorem eos
+        tempore tenetur ea suscipit, quisquam adipisci, tempora sequi nostrum
+        omnis, autem labore eius accusamus quas voluptatibus? Iste odit itaque
+        cumque!
```

Comment:
Why is this text here? 
Isn't the children the actual question?

Reply:
I forgot it there. I added it just to see the width of the answer component. I'll remove it